operation.go

```diff
@@ -224,7 +232,7 @@ func (o *Operation) AddParam(param *Parameter) *Operation {
 // RemoveParam removes a parameter from the operation
 func (o *Operation) RemoveParam(name, in string) *Operation {
 	for i, p := range o.Parameters {
-		if p.Name == name && p.In == name {
+		if p.Name == name && p.In == in {
```

Comment:
PR description says no behavior changes, but this looks like a behavior change.  Does this need to be noted?

Reply:
Yes you're right: I forgot to mention it. I added such minor changes while adding more tests (which did not work as expected, etc..)

Reply:
Done